README.rst

```diff
@@ -108,27 +108,21 @@ first name begins with 'D':
 
 .. code-block:: python
 
-    for user in UserModel.query("John", first_name__begins_with="D"):
+    for user in UserModel.query("John", filter_condition=(UserModel.first_name.startswith("D")):
```

Comment:
I think you should be able to drop the `filter_condition=` part -- at least that was my goal at one point when writing this:
`UserModel.query("John", UserModel.first_name.startswith("D"))`

Reply:
another nit: this example is actually backwards -- `John` should be the range_key and `Denver` should be the hash_key

Reply:
re `filter_condition`: this would require moving the kwarg to an arg too, right? are you suggesting we do that now?

fixed the example for you. it's probably not the most intuitive example we could give here, will consider rewriting

Reply:
it shouldn't -- the way this is supposed to work is that it without the keyword it will get assigned to the `range_key_condition` argument -- then the table will parse it and realize that it's actually a filter_condition since it doesn't match the range_key for the table / index and move it

Reply:
Ah cool, didn't realize that case was handled. I'll update the example